Apromore-Custom-Plugins/CSVImporter-Portal/src/main/java/org/apromore/plugin/portal/csvimporter/CSVFileReader.java

```diff
@@ -46,6 +50,7 @@ CSVReader newCSVReader(Media media, String charset) {
 
             BufferedReader brReader = new BufferedReader(reader);
             String firstLine = brReader.readLine();
+	    brReader.close();
```

Comment:
It'd be slightly better to create `brReader` in a try() block, since the close() call will otherwise be skipped when readLine() throws an IOException